metagraph/src/common/sorted_sets/sorted_set_disk_base.cpp

```diff
@@ -22,7 +22,7 @@ SortedSetDiskBase<T>::SortedSetDiskBase(size_t num_threads,
       merge_count_(merge_count),
       chunk_file_prefix_(tmp_dir/"chunk_"),
       merge_queue_(std::min(reserved_num_elements, QUEUE_EL_COUNT)),
-      async_merge_l1_(merge_count_ == 0 ? 0 : 1, 100) {
+      async_merge_l1_(merge_count_ == 0 ? 0 : 3, 100) {
```

Comment:
why not put a check here to see if `get_num_threads() > 1`?

Reply:
This will be very slow if done strictly with one thread. It's better to have these workers that mostly do slow IO in the background.